<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="frame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_front_color">
          <property name="text">
           <string>Point</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_front_color">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_back_color">
          <property name="text">
           <string>Background</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_back_color">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_hover_color">
          <property name="text">
           <string>Hover</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_hover_color">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_picked_color">
          <property name="text">
           <string>Picked</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_picked_color">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fitting_color">
          <property name="text">
           <string>Fitting</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_fitting_color">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>PointSize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_point_size">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_show_axis">
          <property name="text">
           <string>Show axis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_show_axis">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Label</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButton_load_data">
                 <property name="text">
                  <string>load data</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_start_labeling">
                 <property name="text">
                  <string>start labeling</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_cancel_labeling">
                 <property name="text">
                  <string>cancel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="btn_point_labeling">
               <property name="text">
                <string>point</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_line_labeling">
               <property name="text">
                <string>line</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_plane_labeling">
               <property name="text">
                <string>plane</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_cylinder_labeling">
               <property name="text">
                <string>cylinder</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_check_list">
               <property name="text">
                <string>check label list</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_export_label_infor">
               <property name="text">
                <string>export label information</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Registration</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QPushButton" name="btn_coarse_registration">
               <property name="text">
                <string>Coarse Registration</string>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_fine_registration">
               <property name="text">
                <string>Fine Registration</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_visualization">
               <property name="text">
                <string>Registration Visualization </string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Measurement</string>
          </attribute>
         </widget>
        </widget>
        <widget class="QTextEdit" name="textEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1232</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Adobe Courier</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClean"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu123">
    <property name="title">
     <string>Configuration</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu123"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>&amp;Clean scene</string>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionExport_labeled_info">
   <property name="text">
    <string>Export labeled info</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
